.github/workflows/build.yml

```diff
@@ -32,18 +37,23 @@ jobs:
 
   build-example:
     needs: generate-xcode-project
-    runs-on: macos-15-xlarge
+    runs-on: macos-14-xlarge
     steps:
-      - uses: actions/checkout@v4
+      - name: Set up Xcode
+        uses: maxim-lobanov/setup-xcode@60606e260d2fc5762a71e64e74b2174e8ea3c8bd # v1.6.0
+        with:
+          xcode-version: '16.2'
```

Comment:
When using macOS 14, we must select the Xcode version. Otherwise, it will fail trying to select a version that won't work with our settings.